mysql-test/t/percona_changed_pages_big.test

```diff
@@ -0,0 +1,45 @@
+-- source include/big_test.inc
+-- source include/have_innodb.inc
+
+--echo #
+--echo # PS-5541: Changed page tracking floods the error log with decryption errors
+--echo #
+
+CREATE TABLE `joinit` (
+  `i` int(11) NOT NULL AUTO_INCREMENT,
+  `s` varchar(64) DEFAULT NULL,
+  `t` time NOT NULL,
+  `g` int(11) NOT NULL,
+  PRIMARY KEY (`i`),
+  KEY key_g (`g`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+INSERT INTO joinit VALUES (NULL, uuid(), time(now()),  FLOOR(1 + RAND() * 60));
```

Comment:
Strictly speaking
```suggestion
INSERT INTO joinit VALUES (NULL, uuid(), time(now()), FLOOR(1 + RAND() * 60));
```
still not done but of course not enough to cause a PR respin by itself